Lib/pdb.py

```diff
@@ -570,12 +553,10 @@ def interaction(self, frame, tb_or_exc):
         if isinstance(tb_or_exc, BaseException):
             assert tb is not None, "main exception must have a traceback"
         with self._hold_exceptions(_chained_exceptions):
-            if self.setup(frame, tb):
-                # no interaction desired at this time (happens if .pdbrc contains
-                # a command like "continue")
-                self.forget()
-                return
-            self.print_stack_entry(self.stack[self.curindex])
+            self.setup(frame, tb)
+            # if we have more commands to process, do not show the stack entry
```

Comment:
```suggestion
            # if we do not have more commands to process, do not show the stack entry
```

right?

Reply:
So the logic is "if we do not have more commands, show the stack entry". However, the reason we have this is because we do not want to show the stack entry if we have more commands. The original logic is correct but is double negated from the code.